core/domain/domain.core.yml

```diff
@@ -181,6 +181,9 @@ slots:
   last_has_diff_breathing:
     type: unfeaturized
 
+  invalid_id:
+    type: bool
```

Comment:
invalid_id is too generic... Out of context, it's really hard to know where and why it's used.

Reply:
invalid_reminder_id?